src/components/Form/TextArea/TextArea.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState, ChangeEvent } from 'react';
 import { withA11y } from '@storybook/addon-a11y';
-
+import { textareaCopy } from 'shared/data/copyContent';
 import TextArea from './TextArea';
```

Comment:
I know it's little, but let's make sure there's an empty line between local imports and third-party imports. It's part of our style guide to separate the two groups.